bin/lib/localized-urls.json

```diff
@@ -1,23 +1,68 @@
-        "cards.flyLink": "/pdfs/cards/FlyCards.pdf",
-        "cards.raceLink": "/pdfs/cards/RaceGameCards.pdf",
-        "cards.musicLink": "/pdfs/cards/MusicCards.pdf",
-        "cards.hideLink": "/pdfs/cards/Hide-and-Seek-Cards.pdf",
-        "cards.storyLink": "/pdfs/cards/StoryCards.pdf",
-        "cards.dressupLink": "/pdfs/cards/DressupCards.pdf",
-        "cards.pongLink": "/pdfs/cards/PongCards.pdf",
-        "cards.danceLink": "/pdfs/cards/DanceCards.pdf",
-        "cards.catchLink": "/pdfs/cards/CatchCards.pdf",
-        "cards.petLink": "/pdfs/cards/PetCards.pdf"
+	"cards.starterLink": "/pdfs/cards/Scratch2Cards.pdf",
+	"cards.nameLink": "/pdfs/cards/AnimateYourNameCards.pdf",
+   	"cards.flyLink": "/pdfs/cards/FlyCards.pdf",
+    	"cards.raceLink": "/pdfs/cards/RaceGameCards.pdf",
+    	"cards.musicLink": "/pdfs/cards/MusicCards.pdf",
+    	"cards.hideLink": "/pdfs/cards/Hide-and-Seek-Cards.pdf",
+    	"cards.storyLink": "/pdfs/cards/StoryCards.pdf",
+    	"cards.dressupLink": "/pdfs/cards/DressupCards.pdf",
+    	"cards.pongLink": "/pdfs/cards/PongCards.pdf",
+    	"cards.danceLink": "/pdfs/cards/DanceCards.pdf",
+    	"cards.catchLink": "/pdfs/cards/CatchCards.pdf",
+    	"cards.petLink": "/pdfs/cards/PetCards.pdf",
+	"ttt.MakeItFlyActivityLoc": "/pdfs/cards/FlyCards.pdf",
+    	"ttt.MakeItFlyGuideLoc": "/pdfs/guides/FlyGuide.pdf",
+    	"ttt.AnimateYourNameActivityLoc": "/pdfs/cards/AnimateYourNameCards.pdf",
+    	"ttt.AnimateYourNameGuideLoc": "/pdfs/guides/NameGuide.pdf",
+    	"ttt.MakeMusicActivityLoc": "/pdfs/cards/MusicCards.pdf",
+    	"ttt.MakeMusicGuideLoc": "/pdfs/guides/MusicGuide.pdf",
+    	"ttt.RaceActivityLoc": "/pdfs/cards/RaceGameCards.pdf",
+    	"ttt.RaceGuideLoc": "/pdfs/guides/RaceGuide.pdf",
+    	"ttt.DanceActivityLoc": "/pdfs/cards/DanceCards.pdf",
+    	"ttt.DanceGuideLoc": "/pdfs/guides/DanceGuide.pdf",
+    	"ttt.PongActivityLoc": "/pdfs/cards/PongCards.pdf",
+    	"ttt.PongGuideLoc": "/pdfs/guides/PongGuide.pdf",
+    	"ttt.CatchActivityLoc": "/pdfs/cards/CatchCards.pdf",
+    	"ttt.CatchGuideLoc": "/pdfs/guides/CatchGuide.pdf",
+    	"ttt.HideAndSeekActivityLoc": "/pdfs/cards/Hide-and-Seek-Cards.pdf",
+    	"ttt.HideAndSeekGuideLoc": "/pdfs/guides/Hide-and-Seek-Guide.pdf",
+    	"ttt.VirtualPetActivityLoc": "/pdfs/cards/PetCards.pdf",
+    	"ttt.VirtualPetGuideLoc": "/pdfs/guides/PetGuide.pdf",
+    	"ttt.DressupActivityLoc": "/pdfs/cards/DressupCards.pdf",
+    	"ttt.DressupGuideLoc": "/pdfs/guides/DressupGuide.pdf",
+    	"ttt.StoryActivityLoc": "/pdfs/cards/StoryCards.pdf",
+    	"ttt.StoryGuideLoc": "/pdfs/guides/StoryGuide.pdf"
     },
     "ar": {
         "cards.starterLink": "//cdn.scratch.mit.edu/scratchr2/static/pdfs/help/ar/Scratch2Cards.pdf"
     },
     "ca": {
-        "cards.starterLink": "//cdn.scratch.mit.edu/scratchr2/static/pdfs/help/ca/Scratch2Cards.pdf"
+        "cards.starterLink": "//cdn.scratch.mit.edu/scratchr2/static/pdfs/help/ca/Scratch2Cards.pdf",
```

Comment:
We can't have cards.starterLink multiple times in a single locale, here and line 53. We probably just want to keep the newer one.